app/views/project_information/_project_information_list.erb

```diff
@@ -4,6 +4,10 @@
     row.key { t('project_information.show.project_details.rows.delivery_officer') }
     row.value { @project.delivery_officer&.email or t('project.summary.delivery_officer.unassigned') }
   end
+  summary_list.row do |row|
+    row.key { t('project_information.show.project_details.rows.team_lead') }
```

Comment:
It's a shame the email address get wrapped? Could we use `govuk_summary_list(actions: false)` to stop this? https://govuk-components.netlify.app/components/summary-list/#summary-lists-without-actions